### ISSUE TITLE
Help: 'KafkaClient' object has no attribute 'has_metadata_for_topic'

### ISSUE THREAD

**Issue by ShrCheng** [dpkp/kafka-python#529]
use python-kafka connection kafka

**'KafkaClient' object has no attribute 'has_metadata_for_topic'**

**Comment by dpkp** [comment]
What version are you using and how are you importing KafkaClient?

**Comment by ShrCheng** [comment]
kafka-python-0.9.3

**Comment by dpkp** [comment]
Cannot reproduce:

```
# dpowers at dpowers-macbookpro.local in ~ [20:28:30]
$ virtualenv kafka-0.9.3
New python executable in kafka-0.9.3/bin/python
Installing setuptools, pip...done.

# dpowers at dpowers-macbookpro.local in ~ [20:28:41]
$ cd kafka-0.9.3

# dpowers at dpowers-macbookpro.local in ~/kafka-0.9.3 [20:28:43]
$ source bin/activate

# dpowers at dpowers-macbookpro.local in ~/kafka-0.9.3 (kafka-0.9.3) [20:28:46]
$ pip install kafka-python==0.9.3
Downloading/unpacking kafka-python==0.9.3
  Downloading kafka-python-0.9.3.tar.gz (52kB): 52kB downloaded
  Running setup.py (path:/Users/dpowers/kafka-0.9.3/build/kafka-python/setup.py) egg_info for package kafka-python

Downloading/unpacking six (from kafka-python==0.9.3)
  Downloading six-1.10.0-py2.py3-none-any.whl
Installing collected packages: kafka-python, six
  Running setup.py install for kafka-python

Successfully installed kafka-python six
Cleaning up...

# dpowers at dpowers-macbookpro.local in ~/kafka-0.9.3 (kafka-0.9.3) [20:28:54]
$ python -c "from kafka.client import KafkaClient; assert hasattr(KafkaClient, 'has_metadata_for_topic')"
```

and just to verify my assertion fails on a method that doesn't exist:
```
# dpowers at dpowers-macbookpro.local in ~/kafka-0.9.3 (kafka-0.9.3) [20:32:22]
$ python -c "from kafka.client import KafkaClient; assert hasattr(KafkaClient, 'foobar')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AssertionError
```

**Comment by ShrCheng** [comment]
[root@localhost python_code]# python -c "from kafka.client import KafkaClient; assert hasattr(KafkaClient, 'foobar')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AssertionError

**Comment by dpkp** [comment]
correct -- foobar is not a KafkaClient method (but has_metadata_for_topic is).

**Comment by ShrCheng** [comment]
So, could you tell me how to solve, thank you

**Comment by ShrCheng** [comment]
sorry ， **has_metadata_for_topic**

`[root@localhost python_code]# python -c "from kafka.client import KafkaClient; assert hasattr(KafkaClient, 'has_metadata_for_topic')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AssertionError
`

**Comment by ShrCheng** [comment]
this is python client print kafka help：


>>> help('kafka')

Help on package kafka:

NAME
    kafka

FILE
    /usr/lib/python2.7/site-packages/kafka_python-0.97.0.dev0-py2.7.egg/kafka/__init__.py

PACKAGE CONTENTS
    client
    client_async
    cluster
    codec
    common
    conn
    consumer (package)
    context
    coordinator (package)
    future
    partitioner (package)
    producer (package)
    protocol (package)
    util
    version

CLASSES
    __builtin__.object
        kafka.client.SimpleClient
            KafkaClient
        kafka.conn.BrokerConnection
        kafka.producer.kafka.KafkaProducer
    kafka.consumer.base.Consumer(__builtin__.object)
        kafka.consumer.multiprocess.MultiProcessConsumer
...

**Comment by dpkp** [comment]
Ok, you are using a version installed directly from github master
("0.97.0.dev") and not the released version 0.9.3.

Beware that the current master branch is breaking some interfaces and
deprecating others.

For your purposes, try using SimpleClient instead:

```

from kafka.client import SimpleClient

```

**Comment by ShrCheng** [comment]
Ok, thanks, after upgrading to solve
